src/Controller/EnMarche/Jecoute/JecouteRegionController.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace App\Controller\EnMarche\Jecoute;
+
+use App\Controller\EnMarche\AccessDelegatorTrait;
+use App\Entity\Jecoute\Region;
+use App\Form\Jecoute\RegionType;
+use App\Jecoute\RegionManager;
+use App\Repository\Geo\RegionRepository as GeoRegionRepository;
+use App\Repository\Jecoute\RegionRepository;
+use Doctrine\Common\Persistence\ObjectManager;
+use Ramsey\Uuid\Uuid;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @Route("/espace-candidat/campagne", name="app_jecoute_candidate_region_")
```

Comment:
what about naming it `/espace-candidat/region`?

Reply:
It will not be clear to users.
`campagne` corresponds to the name of the page/in the menu.